l2-bridge-linea/src/utils/events/bridge_events.ts

```diff
@@ -0,0 +1,89 @@
+    alertId: 'L2-BRIDGE-ROLE-ADMIN-CHANGED',
+    name: '🚨 Linea L2 Bridge: Role Admin changed',
+    description: (args: Result) =>
+      `Role Admin for role ${args.role}(${RolesAddrToNameMap.get(args.role) || 'unknown'}) ` +
+      `was changed from ${args.previousAdminRole} to ${args.newAdminRole}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Info,
+  },
+  {
+    address: L2_ERC20_TOKEN_GATEWAY_ADDRESS,
+    event: 'event RoleGranted(bytes32 indexed role, address indexed account, address indexed sender)',
+    alertId: 'L2-BRIDGE-ROLE-GRANTED',
+    name: '⚠️ Linea L2 Bridge: Role granted',
+    description: (args: Result) =>
+      `Role ${args.role}(${RolesAddrToNameMap.get(args.role) || 'unknown'}) ` +
+      `was granted to ${args.account} by ${args.sender}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Info,
+  },
+  {
+    address: L2_ERC20_TOKEN_GATEWAY_ADDRESS,
+    event: 'event RoleRevoked(bytes32 indexed role, address indexed account, address indexed sender)',
+    alertId: 'L2-BRIDGE-ROLE-REVOKED',
+    name: '⚠️ Linea L2 Bridge: Role revoked',
+    description: (args: Result) =>
+      `Role ${args.role}(${RolesAddrToNameMap.get(args.role) || 'unknown'}) ` +
+      `was revoked to ${args.account} by ${args.sender}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Info,
+  },
+  {
+    address: L2_ERC20_TOKEN_GATEWAY_ADDRESS,
+    event: 'event DepositsEnabled(address indexed enabler)',
+    alertId: 'L2-BRIDGE-DEPOSITS-ENABLED',
+    name: '✅ Linea L2 Bridge: Deposits Enabled',
+    description: (args: Result) => `Deposits were enabled by ${args.enabler}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Info,
+  },
+  {
+    address: L2_ERC20_TOKEN_GATEWAY_ADDRESS,
+    event: 'event DepositsDisabled(address indexed disabler)',
+    alertId: 'L2-BRIDGE-DEPOSITS-DISABLED',
+    name: '❌ Linea L2 Bridge: Deposits Disabled',
+    description: (args: Result) => `Deposits were disabled by ${args.disabler}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Info,
+  },
+  {
+    address: L2_ERC20_TOKEN_GATEWAY_ADDRESS,
+    event: 'event WithdrawalsEnabled(address indexed enabler)',
+    alertId: 'L2-BRIDGE-WITHDRAWALS-ENABLED',
+    name: '✅ Linea L2 Bridge: Withdrawals Enabled',
+    description: (args: Result) => `Withdrawals were enabled by ${args.enabler}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Info,
+  },
+  {
+    address: L2_ERC20_TOKEN_GATEWAY_ADDRESS,
+    event: 'event WithdrawalsDisabled(address indexed disabler)',
+    alertId: 'L2-BRIDGE-WITHDRAWALS-DISABLED',
+    name: '❌ Linea L2 Bridge: Withdrawals Disabled',
+    description: (args: Result) => `Withdrawals were disabled by ${args.enabler}`,
+    severity: FindingSeverity.High,
+    type: FindingType.Info,
+  },
+  {
+    address: L2_ERC20_TOKEN_GATEWAY_ADDRESS,
+    event: 'event Initialized(address indexed admin)',
```

Comment:
```suggestion
    event: 'event Initialized(uint8 version)',
```
see: https://lineascan.build/address/0x2bfdf4a0d54c93a4baf74f8dcea8a275d8ee97a9#code#F4#L73

Reply:
added